src/relay/backend/graph_runtime_codegen.cc

```diff
@@ -639,6 +668,35 @@ class GraphRuntimeCodegenModule : public runtime::ModuleNode {
       return PackedFunc([sptr_to_self, this](TVMArgs args, TVMRetValue* rv) {
         *rv = this->output_.lowered_funcs;
       });
+    } else if (name == "get_external_funcs") {
```

Comment:
I don't really like the current monolithic approach to handle the external compilations. Perhaps we can think a bit deeper.